tests/unit/test_run.py

```diff
@@ -2597,15 +2597,15 @@ def test_long_data_time_axis(scm_run, time_axis, mod_func):
 
 
 @time_axis_checks
-@patch("scmdata.plotting.sns.lineplot")
```

Comment:
Any reason why you prefer the inline patch calls over the decorators?

Reply:
I couldn't get the importorskip function to work properly using the decorators (it would explode before realising it should have skipped because seaborn isn't installed). Maybe if I'd put importorskip as the first decorator it might have worked...